tests/functional/pv/test_qos_limits.py

```
@@ -0,0 +1,568 @@
+            for tmp_file in [tmp_silver, tmp_gold]:
+                if os.path.exists(tmp_file):
+                    os.remove(tmp_file)
+
+        request.addfinalizer(cleanup)
+
+    @pytest.fixture
+    def test_resources_cleanup(self, request):
+        """Maintains low-impact cleanup handles for dynamically generated pods and claims."""
+        resources = {"pods": [], "pvcs": []}
+
+        def resource_teardown():
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `resource_teardown`

Add return type annotation: `None`

(ANN202)